### PR TITLE
#1469 Implement topics page

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1469)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      );
+
+      setState({
+        topics: orderBy(topics, 'name'),
+        brokerClusterName: topicsUnderBrokerCluster,
+      });
+    }
+  };
+
+  useEffect(() => {
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: `const fetchData = useCallback(async () => {
       await fetchWorker();
       await fetchTopics();
       setState({ isLoading: false });
     },[props]);

useEffect(() => {
     fetchData();
   }, [fetchData]);`

Maybe this is better?

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    isDeleting: false,
+    isNewModalOpen: false,
+    isDeleteModalOpen: false,
+    topics: [],
+    topicToBeDeleted: '',
+    brokerClusterName: '',
+  });
+
+  const fetchWorker = async () => {
+    const workerName = props.match.url.split('/')[2]; // use params instead of manually splitting the url name
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: Maybe unite workerName -> workspaceName

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: I'm not sure we need to use `useCallback`. It's supposed to enhance performance. Does this work on your end?

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: Not sure about this. As worker === workspace and our fetch API is using fetch worker...

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: I opened another issue to track this. I think there should be better ways to handle async data fetching with react hooks. Let's leave it as is and move on the last page - overview

Comment (konekoya) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: #1538

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Topics/Topics.js]: We can father discuss this as the workspace is not a real service in the backend and only lives in the frontend right now :)

**Pull request merged**